bundler/spec/support/helpers.rb

```diff
@@ -25,11 +25,7 @@ def reset_paths!
 
     def self.bang(method)
       define_method("#{method}!") do |*args, &blk|
-        send(method, *args, &blk).tap do
-          unless last_command.success?
-            raise "Invoking #{method}!(#{args.map(&:inspect).join(", ")}) failed:\n#{last_command.stdboth}"
-          end
-        end
+        send(method, *args, &blk)
```

Comment:
What's purpose of this method now?

Reply:
Oh, this one used to implement the raising `!` behaviour. Now it does nothing other than delegating, so that `bundle!` and `bundle` now do the same thing.

I will remove this method in the next PR and remove all usages of "hang helpers". But at the last minute I decided to leave that part for the next PR because the diff was becoming huge.

Reply:
Do you prefer that I remove it in this PR?

Reply:
I'll merge this and remove this method in the next PR as I planned to :+1:.